app/forms/flood_risk_engine/steps/base_form.rb

```diff
@@ -19,6 +19,12 @@ def redirect?
         false
       end
 
+      # Default to the url as defined in the view/partial but provide opportunity for Forms to
+      # wire up the continue button as they see require
+      def url
```

Comment:
I'm not sure being able to change the continue button is necessary or desirable, but if it is the name should be meaningful. For example: `continue_button_url`